packages/feedback/src/screenshot/components/ScreenshotEditorv2.tsx

```diff
@@ -0,0 +1,356 @@
+      if (currentRect) {
+        drawScene();
+      }
+    }, [currentRect]);
+
+    function drawBuffer(): void {
+      const ctx = imageBuffer.getContext('2d', { alpha: false });
+      const measurementDiv = measurementRef.current;
+      if (!imageBuffer || !ctx || !imageSource || !measurementDiv) {
+        return;
+      }
+
+      ctx.drawImage(imageSource, 0, 0);
+
+      const grayWashBufferBig = DOCUMENT.createElement('canvas');
+      grayWashBufferBig.width = imageBuffer.width;
+      grayWashBufferBig.height = imageBuffer.height;
+
+      const grayCtx = grayWashBufferBig.getContext('2d');
+      if (!grayCtx) {
+        return;
+      }
+
+      // applies the graywash if there's any boxes drawn
+      if (drawCommands.length || currentRect) {
+        grayCtx.fillStyle = 'rgba(0, 0, 0, 0.25)';
+        grayCtx.fillRect(0, 0, imageBuffer.width, imageBuffer.height);
+      }
+
+      drawCommands.forEach(rect => {
+        drawRect(rect, grayCtx);
+      });
+      ctx.drawImage(grayWashBufferBig, 0, 0);
+    }
+
+    function drawScene(): void {
+      const graywashCanvas = graywashRef.current;
+      if (!graywashCanvas) {
+        return;
+      }
+
+      const ctx = graywashCanvas.getContext('2d');
+      if (!ctx) {
+        return;
+      }
+
+      ctx.clearRect(0, 0, graywashCanvas.width, graywashCanvas.height);
+
+      // applies the graywash if there's any boxes drawn
+      if (drawCommands.length || currentRect) {
+        ctx.fillStyle = 'rgba(0, 0, 0, 0.25)';
+        ctx.fillRect(0, 0, graywashCanvas.width, graywashCanvas.height);
+      }
+
+      const scale = graywashCanvas.clientWidth / imageBuffer.width;
+      drawCommands.forEach(rect => {
+        drawRect(rect, ctx, scale, 2);
+      });
+
+      if (currentRect) {
+        drawRect(currentRect, ctx, 1, 2);
+        setCurrentRect(undefined);
+      }
+    }
+
+    useTakeScreenshot({
+      onBeforeScreenshot: hooks.useCallback(() => {
+        (dialog.el as HTMLElement).style.display = 'none';
+        setdisplayEditor(false);
+      }, []),
+      onScreenshot: hooks.useCallback((imageSource: HTMLVideoElement) => {
+        const bufferCanvas = DOCUMENT.createElement('canvas');
+        bufferCanvas.width = imageSource.videoWidth;
+        bufferCanvas.height = imageSource.videoHeight;
+        bufferCanvas.getContext('2d', { alpha: false })?.drawImage(imageSource, 0, 0);
+        setimageSource(bufferCanvas);
+
+        imageBuffer.width = imageSource.videoWidth;
+        imageBuffer.height = imageSource.videoHeight;
+      }, []),
+      onAfterScreenshot: hooks.useCallback(() => {
+        (dialog.el as HTMLElement).style.display = 'block';
+        setdisplayEditor(true);
+      }, []),
+      onError: hooks.useCallback(error => {
+        (dialog.el as HTMLElement).style.display = 'block';
+        setdisplayEditor(true);
+        onError(error);
+      }, []),
+    });
+
+    const onDraw = (e: MouseEvent): void => {
```

Comment:
```suggestion
    const handleMouseDown = (e: MouseEvent): void => {
```

consistent event callback names 1/3